### PR TITLE
automation: Minor fix in authentication docs

### PR DESCRIPTION
**Pull request by dachiefjustice**
## Overview
While playing with ZAP's authentication framework, I found a probable typo/brain-o at https://www.zaproxy.org/docs/desktop/addons/automation-framework/authentication/#configuring-authentication

It currently reads:
> Then you can create a new Authentication Framework job using the context(s) you have tested

I think it should read:
> Then you can create a new Automation Framework job using the context(s) you have tested

It looks like this content is generated from this add-on docs, so I submitted the PR here (assuming it will find its way to the published docs site).

## Related Issues
None

## Checklist
N/A -- doc-only update

### CODE REVIEW

File: addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/authentication.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <H2>Configuring Authentication</H2>
 
 The recommended way to configure authentication is to do so via the ZAP desktop GUI - 
 this gives you complete control over all aspects and allows you to test it in place.<br>
-Then you can create a new Authentication Framework job using the context(s) you have tested - 
+Then you can create a new Automation Framework job using the context(s) you have tested -
```
Comment (thc202) [inline addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/authentication.html:46]: I think it's also job → plan

**Comment by thc202** [comment]
Worth updating the changelog.

**Comment by psiinon** [comment]
Oops, my bad 😛

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**